superset/sql_parse.py

```diff
@@ -0,0 +1,90 @@
+import sqlparse
+from sqlparse.sql import IdentifierList, Identifier
+from sqlparse.tokens import Keyword, DML, Name
+
+
+RESULT_OPERATIONS = {'UNION', 'INTERSECT', 'EXCEPT'}
+PRECEDES_TABLE_NAME = {'FROM', 'JOIN', 'DESC', 'DESCRIBE', 'WITH'}
+
```

Comment:
I want to keep it as simple as I can for now. Happy to redesign it once we have other use cases.